plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/webview/Browser.kt

```diff
@@ -32,6 +33,10 @@ class Browser(parent: Disposable) {
         loadWebView(isCodeTransformAvailable, isFeatureDevAvailable)
     }
 
+    override fun dispose() {
+        Disposer.dispose(jcefBrowser)
+    }
+
```

Comment:
isnt it already parented to a disposable

Reply:
yea, the parent here is `AmazonQToolWindow`, but here we only want to dispose this jcefBrowser and its down-stream?

Reply:
then shouldnt it be reparented and we can drop the parent param?

Reply:
So remove the parent param and while registering the disposer send the current browser instance?

Reply:
yea, we should just do 
```
val jcefBrowser = createBroowser(this)
```
and we can remove parent disposable from ctor